doc/UsersGuide/source/CAPE2020.rst

```diff
@@ -0,0 +1,166 @@
+* Set ``CRES_HIRES`` to desired resolution for coldstart files (e.g., ``C192``)
+
+In the machine-specific driver (e.g., ``driver.jet.sh``), users will need to set ``PROJECT_CODE`` to an account that the user can use for submitting batch jobs. 
+
+After configuring the utility, users can run the driver script: 
+
+.. code-block:: console
+   
+   ./driver.<platform>.sh
+
+These steps will (1) pull raw GFS data from HPSS into ``EXTRACT_DIR`` and then (2) convert the raw data to coldstart ICs (placed in ``OUTDIR/gfs.YYYYMMDD/HH/model_data/atmos/input``). The converted cold start ICs can then be used as input data for certain UFS WM regression tests (RTs) of corresponding model resolution and configuration. 
+
+To use the converted coldstart files with the UFS WM, copy the ``gfs*.nc`` and ``sfc*.nc`` files from ``OUTDIR/gfs.YYYYMMDD/HH/model_data/atmos/input`` into the ``INPUT`` directory of the UFS WM RT run directory, and adjust the model start time in the ``model_configure`` file. Note that the resolution of the converted cold start GFS data must match the ATM resolution of the UFS WM RT (e.g., use C48 GFS cold start data in the ``control_c48.v2.sfc`` RT). Then, edit the model start year, month, day, and hour in the RT ``model_configure`` file. For example: 
+
+.. code-block::
+
+   start_year:              2021
+   start_month:             03
+   start_day:               22
+   start_hour:              06
+   start_minute:            0
+   start_second:            0
+   nhours_fcst:             24
+   fhrot:                   0
+   ...
+
+The user can then run the ``sbatch job_card`` command to rerun ``fv3.exe``. The case should now run at the new model start time with the coldstart ICs. Users with access to Jet may view an example RT at ``/lfs4/HFIP/hfv3gfs/Cameron.Book/RT_RUNDIRS/Cameron.Book/FV3_RT/rt_1826477/control_c48.v2.sfc_intel``, which used coldstart ICs generated through ``gdas_init``/``chgres_cube`` for the 2020 CAPE case. The input data can be viewed at ``/lfs4/HFIP/hfv3gfs/Cameron.Book/chgres_data/converted/gfs.20200723/00/model_data/atmos/input``:
+
+.. code-block:: console
+
+   gfs_ctrl.nc	   gfs_data.tile4.nc  sfc_data.tile2.nc  sfc_data.tile6.nc
+   gfs_data.tile1.nc  gfs_data.tile5.nc  sfc_data.tile3.nc
+   gfs_data.tile2.nc  gfs_data.tile6.nc  sfc_data.tile4.nc
+   gfs_data.tile3.nc  sfc_data.tile1.nc  sfc_data.tile5.nc
+
+.. note:: 
+
+   Note that since 6/4/24, the ``develop`` branch of UFS_UTILS generates only version 2 (v2) surface (sfc) files via ``gdas_init`` and ``chgres_cube``. Therefore, successful integration of the converted cold start files has only been achieved using the recently added ``v2.sfc`` WM RTs (see e.g., :wm-repo:`control_c48.v2.sfc <blob/develop/tests/tests/control_c48.v2.sfc>` and PRs :wm-repo:`#2005 <pull/2005>` and :wm-repo:`#1977 <pull/1977>`). Since the v2 surface files are significantly different from the v1 surface file format, v2 files do not work with non-v2.sfc RTs.
```

Comment:
i would change " Since the v2 surface files are significantly different from the v1 surface file format, v2 files do not work with non-v2.sfc RTs."   to something like

Since the v2 surface files are significantly different from the v1 surface file format, the user may need to re-configure the higher resolution test case to ensure that fix files and other input data used are consistent with v2 surface files.